### ISSUE TITLE
Implement user settings

### ISSUE THREAD

**Issue by toastedtruth** [SmartPaws/smartpaws-android#6]
Settings button on the navigation drawer currently does nothing. 

State any settings that we could allow the user to change in this issue.

**Comment by Crazymoomin** [comment]
I would like to see a boolean flag that allows the app to automatically check for updates to the schedule on main activity start. This might also be useful for future features, that allow a receiver to force the app to update (In the event of a schedule change for example).

**Comment by toastedtruth** [comment]
With something like this, would it be best to leave it disabled by default? Some people are limited on data connectivity, especially if people are travelling from abroad.

**Comment by Crazymoomin** [comment]
Yes, I think that's best. In a future version (maybe not v1.0) I'd like this flag to be forced on when a schedule change notification is sent (this could probably be disabled some other way too, if the user doesn't want to be informed of schedule changes for example, which would need a data connection anyway. We could probably have some sort of "Offline Mode" that does this).

**Comment by toastedtruth** [comment]
According to Delph (CFz events), schedule changes are very rarely - if ever - uploaded to their system. They're usually posted on Twitter. Right now I wouldn't be too worried if it was disabled by default, but enabled in a future release

**Comment by Crazymoomin** [comment]
Yeah, hopefully if/when the ConOps dashboard gets implemented, that may change. But that's for the future. :-)

**Comment by toastedtruth** [comment]
ccc87ded2b3432073ae794417cefcfa75e0d8073
